FoodPin/FoodPin/View/RestaurantDetailTwoColumnCell.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  RestaurantDetailTwoColumnCell.swift
+//  FoodPin
+//
+//  Created by Andreea Sburlea on 28.04.2022.
+//
+
+import UIKit
+
+class RestaurantDetailTwoColumnCell: UITableViewCell {
+
+    @IBOutlet var column1TitleLabel: UILabel! {
+        didSet {
+            column1TitleLabel.text = column1TitleLabel.text?.uppercased()
+            column1TitleLabel.numberOfLines = 0
+        }
+    }
+
+    @IBOutlet var column1TextLabel: UILabel! {
+        didSet {
+            column1TextLabel.numberOfLines = 0
+        }
+    }
+
+    @IBOutlet var column2TitleLabel: UILabel! {
+        didSet {
+            column2TitleLabel.text = column2TitleLabel.text?.uppercased()
+            column2TitleLabel.numberOfLines = 0
+        }
+    }
+
+    @IBOutlet var column2TextLabel: UILabel! {
+        didSet {
+            column2TextLabel.numberOfLines = 0
+        }
+    }
```

Comment:
Same as the previous comments and the text can be made uppercased directly where you initially set it